src/common/images/thumbnailImage.component.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+import { APIImage } from '../../api/api.types';
+
+export interface ThumbnailImageProps {
+  onClick?: () => void;
+  image: APIImage;
+}
+
+const ThumbnailImage = (props: ThumbnailImageProps) => {
+  const { onClick, image } = props;
+  const [hasError, setHasError] = React.useState<string | undefined>(undefined);
+  return (
+    <>
+      {hasError === image.id ? (
```

Comment:
Just an observation, if the image is invalid it flickers
![60](https://github.com/user-attachments/assets/1b2317cc-e7d4-4b3c-ae06-5a46647b1be2)
![61](https://github.com/user-attachments/assets/fc0aa5b7-4318-4378-a8bc-feac29d10938)

Is there a reason the two are different rather than using the same for both?

Reply:
how did you trigger this

Reply:
I manually set my thumbnail to "test" in the database, I assume it doesn't set the error state until its rendered once and then it changes.

Reply:
Yes as I said it just flickers, I also asked `Is there a reason the two are different rather than using the same for both?`.